src/app/public/modules/progress-indicator/progress-indicator-status-marker/progress-indicator-status-marker.component.html

```diff
@@ -0,0 +1,18 @@
+<div
+  class="sky-progress-indicator-status-marker"
+  role="presentation"
```

Comment:
This is good feedback. Since this pull request is only making the previous screen-reader behavior correct (before, the elements were purely being used as presentation), I think it deserves an audit by @Blackbaud-MattGregg, and perhaps a new issue to track that work effort.

Reply:
`role="presentation"` will tell screen readers this is purely layout and there is nothing meaningful in this particular markup. Is that necessarily true? We have indications of status here, which is the whole point of the component. Plus, I'm not seeing any text-equivalent for the icon below. Should we rethink this a bit?

I wonder if a better fit would be using something like `role="status"`, and then adding a text equivalent for the icon below. We could potentially break this out into a separate piece of work. What do you think?